xr_sample_project/Assets/SampleViewer/Samples/VRSample/Scripts/ContinuousMovement.cs

```diff
@@ -18,6 +18,7 @@ public class ContinuousMovement : MonoBehaviour
     private CharacterController controller;
```

Comment:
For each code file in this sample, remove the unnecessary `using` statements and run DotNet format.